pkg/resources/resource_monitor.go

```diff
@@ -77,6 +77,13 @@ var resourceMonitorSchema = map[string]*schema.Schema{
 		Default:     false,
 		ForceNew:    true,
 	},
+	"warehouses": {
+		Type:		schema.TypeSet,
+		Optional: true,
+		Description: "A list of warehouses to apply the resource monitor to.",
+		Elem:		&schema.Schema{Type: schema.TypeString},
+		ForceNew: true,
+	},
```

Comment:
Could you also run `make fmt` to fix the spacing here.

Reply:
okay done